apps/consent-ui/src/components/views/Invite/ClinicianInviteForm/index.tsx

```diff
@@ -274,7 +276,7 @@ const ClinicianInviteFormComponent = ({
 								<>
 									<p className={styles.assentFormAcknowledged}>
 										{textDict.assentFormAcknowledgedText1}{' '}
-										<Link href="" prefetch={false}>
+										<Link href={assentFormPdfUrl} prefetch={false}>
```

Comment:
Let's make this PDF open in a few tab so the Clinician doesn't lose anything they may have already entered into the invite form when they click this link:
```suggestion
										<Link href={assentFormPdfUrl} prefetch={false} target="_blank">
```

Reply:
thanks for pointing this out. we have it on another PDF link as well.